api/src/main/scala/hmda/api/http/institutions/UploadPaths.scala

```diff
@@ -67,12 +67,11 @@ trait UploadPaths extends InstitutionProtocol with ApiErrorProtocol with Submiss
       }
     }
 
-  private def checkSubmissionIsCreated(submissionsActor: ActorRef, submissionId: SubmissionId)(implicit ec: ExecutionContext): Future[Boolean] = {
-    val submission = (submissionsActor ? GetSubmissionById(submissionId)).mapTo[Submission]
-    submission.map(_.status == Created)
+  private def checkSubmissionIsCreated(submissionsActor: ActorRef, submissionId: SubmissionId)(implicit ec: ExecutionContext): Future[Submission] = {
```

Comment:
Since this function is now just returning a `Future[Submission]` it probably shouldn't be called `checkSubmissionIsCreated`. It might also just be worthwhile to just remove it all together and just put its logic in the `for` block since it's only the one line.